### PR TITLE
implement new interface in Jenkinsfile

### PR DESCRIPTION
**Pull request by pablodcar**
Changing stolon’s Jenkinsfile/Makefile to work in others' Jenkins with more parameters.

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -107,12 +136,35 @@ make build-app OPS_URL=$OPS_URL"""
...
       }
     }
+
+    stage('import') {
+      if (params.IMPORT_APP) {
+        withCredentials([
+          string(credentialsId: params.OPS_CENTER_CREDENTIALS, variable: 'API_KEY'),
+        ]) {
+          withEnv(MAKE_ENV) {
+            sh 'make import'
```
Comment (UnderGreen) [inline Jenkinsfile:154]: `make import` is working incorrectly here. I am going to add another make target to build app and provide you guidance on how to use that.

File: Makefile [hunk 2]
```diff
@@ -52,8 +51,7 @@ IMPORT_OPTIONS := --vendor \
 		--registry-url=leader.telekube.local:5000 \
 		$(IMPORT_IMAGE_OPTIONS)
 
-TELE_BUILD_OPTIONS := --insecure \
-		--repository=$(OPS_URL) \
+TELE_BUILD_OPTIONS := --repository=$(OPS_URL) \
```
Comment (UnderGreen) [inline Makefile:54]: Why did you delete `--insecure` flag here?

Comment (pablodcar) [reply Makefile:54]: Good question, IMO it should be passed as `EXTERNAL_GRAVITY_OPTIONS` or every command should have `--insecure`. If not we're passing `--insecure` with `EXTERNAL_GRAVITY_OPTIONS` for _some_ commands and using an implicit `--insecure` for others and we have to use different env variables to don't pass `--insecure` twice.

File: Jenkinsfile (hunk 1 above)
Comment (pablodcar) [reply Jenkinsfile:154]: Ok, I'll wait for the final target to do the changes in the other projects.

**Comment by wadells** [comment]
Out of curiosity (as a robotest maintainer): What about the upgrade tests motivated this special casing?

**Comment by UnderGreen** [comment]
@wadells this upgrade is for gravity 7.0.x, which does not support yet param `--upgrade-via` and there is no way to test upgrades from 5.5.
@pablodcar can you change in this PR default gravity version to 7.0.12?

**Comment by pablodcar** [comment]
> @pablodcar can you change in this PR default gravity version to 7.0.12?

done
